CompetitionTemplate/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -0,0 +1,60 @@
+package competition.subsystems.elevator;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+
+
+@Singleton
+public class ElevatorSubsystem extends BaseSubsystem {
+	
```

Comment:
Let's add a DoubleProperty that keeps track of the default elevator power.

Reply:
Sounds Good!